src/com/trollworks/gcs/advantage/Advantage.java

```diff
@@ -1040,6 +1040,21 @@ public boolean setModifiers(List<Modifier> modifiers) {
         return false;
     }
 
+    /**
+     * Find the first modifier that matches 'name'.  
+     *
+     * @param name case insensitive
+     * @return
```

Comment:
@return by itself isn't useful. Please change this to match other uses... something like:
```
/**
 * @param name The name to match against. Case-insensitive.
 * @return The first modifier that matches the name.
 */